.travis.yml

```diff
@@ -13,25 +13,55 @@ notifications:
     recipients:
        - dave@dav.ee
 env:
+  global: # default values
+    - ROS_DISTRO=melodic
+    - ROS_REPO=ros
+    - BEFORE_DOCKER_SCRIPT="echo Performing normal build"
+    - EXPECTED_RESULT=0
```

Comment:
This new feature is clunky and a bit counter-intuitive. You've added it to enable testing of the testing framework, correct? At least, please provide ample commenting to what it does.

Reply:
Are you referring to `EXPECTED_RESULT`?
Yes this was needed for unit testing of moveit_ci. I came up with a different approach now.

Or are you referring to `global` variable definitions? This is standard in Travis.